updatable_timer/workflow.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime, timezone
+from typing import Optional
+
+from temporalio import workflow
+
+from updatable_timer.updatable_timer_lib import UpdatableTimer
+
+
+@workflow.defn
+class Workflow:
+    def __init__(self):
```

Comment:
Recommend return type annotations everywhere

Reply:
My Java brain protested this for a constructor :).